e2e/helpers/pages/admin/sidebar/sidebar-page.ts

```diff
@@ -0,0 +1,73 @@
+import {AdminPage} from '@/admin-pages';
+import {Locator, Page} from '@playwright/test';
+
+/**
+ * SidebarPage uses semantic, accessibility-first locators.
+ * This approach tests the UI as users interact with it, not implementation details.
+ *
+ * Accessibility features:
+ * ✓ Active nav links have aria-current="page"
+ * ✓ Posts toggle button has aria-expanded
+ */
+export class SidebarPage extends AdminPage {
+    public readonly sidebar: Locator;
+    public readonly postsToggle: Locator;
+    public readonly userDropdownTrigger: Locator;
+    public readonly nightShiftToggle: Locator;
+    public readonly whatsNewButton: Locator;
+    public readonly userProfileLink: Locator;
+    public readonly signOutLink: Locator;
+
+    constructor(page: Page) {
+        super(page);
+        this.sidebar = page.getByRole('navigation');
+        this.postsToggle = this.sidebar.getByRole('button', {name: /toggle post views/i});
+        this.userDropdownTrigger = page.locator('[data-test-nav="arrow-down"]');
+        this.nightShiftToggle = page.getByRole('button', {name: /dark mode/i}).or(page.getByRole('menuitem', {name: /dark mode/i}).getByRole('switch'));
```

Comment:
`nit` since we use `or` for locating here, will this always work, will it find correct locators (I assume so just a double check here)